docs/pages/blog/2023-discord-announcement.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'MUI is now on Discord!'
+description: Come join our community to engage in lively discussions, share your projects, and interact with the MUI team
+date: 2023-08-02T00:00:00.000Z
+authors: ['richbustos']
+tags: ['News']
+card: true
+---
+
+<a href="https://mui.com/r/discord"><img src="/static/blog/2023-discord-announcement/discord.png" width="1200" height="640" style="margin-bottom: 16px;" alt="Discord banner with link" /></a>
```

Comment:
The width and height are wrong, there is a layout shift when opening the page. The best practice in our case is to use the exact image dimension `width="2560" height="1280"` at least to have the same ratio.

Reply:
The exact image is`width="1280" height="640"` now.